sql/core/src/test/resources/test-data/cars-crlf.csv

```diff
@@ -0,0 +1,7 @@
+
+year,make,model,comment,blank
+"2012","Tesla","S","No comment",
+
+1997,Ford,E350,"Go get one now they are going fast",
+2015,Chevy,Volt
+

```

Comment:
just for clarification, this file has newline variant combinations, right?

Reply:
Yea, if you open it with `vi -b sql/core/src/test/resources/test-data/cars-crlf.csv`, you can see the `^M` characters which according to [this](https://stackoverflow.com/questions/3860519/see-line-breaks-and-carriage-returns-in-editor) represents a CLRF.